recipes/expat/all/conanfile.py

```diff
@@ -81,13 +94,29 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
+    @property
+    def _library_name(self):
+        prefix = ""
+        postfix = ""
+        if self.settings.os == "Windows":
+            if tools.Version(self.version) >= "2.2.10" and self.options.get_safe("char_type") != "char":
+                postfix += "w"
+            if self.settings.build_type == "Debug":
+                postfix += "d"
+        if self.settings.compiler == "Visual Studio":
+            if tools.Version(self.version) > "2.2.7":
+                postfix += "MT" if "MT" in self.settings.compiler.runtime else "MD"
+        if self.settings.compiler == "Visual Studio":
+            prefix = "lib"
+        return prefix + "expat" + postfix
```

Comment:
Is it worth the effort?

Reply:
Yes/no, but I give up for now :)

Reply:
Actually, if shared, runtime postfix is not added: https://github.com/libexpat/libexpat/blob/a28238bdeebc087071777001245df1876a11f5ee/expat/CMakeLists.txt#L328

Welcome in hell.